ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -161,6 +170,31 @@ func (scbuilder *SiteConfigBuilder) getClusterCR(clusterId int, siteConfigTemp S
 	return mapIntf, nil
 }
 
+func (scbuilder *SiteConfigBuilder) getBiosConfigMapCR(clusterId int, siteConfigTemp SiteConfig, nodeId int) (interface{}, error) {
+
+	// if BiosConfigMapRef is not set for the host, use the one set for the cluster
+	if siteConfigTemp.Spec.Clusters[clusterId].Nodes[nodeId].BiosConfigMapRef.Name == "" {
+		siteConfigTemp.Spec.Clusters[clusterId].Nodes[nodeId].BiosConfigMapRef = siteConfigTemp.Spec.Clusters[clusterId].BiosConfigMapRef
+	}
+
+	annotations := map[string]string{"argocd.argoproj.io/sync-wave": "1"}
```

Comment:
We should define all of the waves as constants and keep those constants together in one place so that it is easy to look at all mappings of CRs to waves in one place.

Reply:
Defining all the waves as constants will be addressed in a followup PR.